examples/visual-tests/amp-story/info-dialog.html

```diff
@@ -63,20 +64,41 @@
         poster-portrait-src="/examples/visual-tests/picsum.photos/image981_900x1600.jpg"
         poster-landscape-src="/examples/visual-tests/picsum.photos/image981_1600x900.jpg"
         poster-square-src="/examples/visual-tests/picsum.photos/image981_1600x1600.jpg">
+        <amp-analytics id="analytics1">
+            <script type="application/json">
+              {
+                "requests": {
+                  "endpoint": "https://raw.githubusercontent.com/ampproject/amphtml/master/examples/img/ampicon.png",
+                  "base": "${endpoint}?${type|default:foo}&path=${canonicalPath}",
+                  "pageView": "${base}?&index=${storyPageIndex}&count=${storyPageCount}&id=${storyPageId}&muted=${storyIsMuted}&progress=${storyProgress}",
+                  "click": "${base}?&index=${storyPageIndex}&id=${storyPageId}&bookendTargetHref=${storyBookendTargetHref}&bookendCardType=${storyBookendComponentType}&bookendCardPosition=${storyBookendComponentPosition}"
+                },
+                "triggers": {
+                  "trackInfoDialogOpen": {
+                    "on": "story-open",
+                    "tagName": "amp-story-info-dialog",
+                    "request": "base"
+                  },
+                  "trackInfoDialogExit": {
+                    "on": "story-close",
+                    "tagName": "amp-story-info-dialog",
+                    "request": "base"
+                  }
+                }
+              }
+            </script>
+          </amp-analytics>
 
       <amp-story-page id="cover">
         <amp-story-grid-layer template="fill">
-          <amp-video autoplay layout="fill" poster="/examples/visual-tests/picsum.photos/image981_1600x900.jpg">
-          </amp-video>
+          <amp-img layout="fill" src="/examples/visual-tests/picsum.photos/image981_1600x900.jpg">
```

Comment:
@gmajoulet can you take a look at the visual diff test? I just changed this example to use an amp-img instead of an `amp-video` that was missing a source and only had a poster.

Reply:
Oh cool, thanks for the fix :) LGTM